pkg/config/config_policy_routing.go

```diff
@@ -254,13 +255,25 @@ func init() {
 			RuleList: netlink.RuleList,
 		},
 	}
+	glog.Info("Including local table rules.")
+	PolicyRoutingConfigSet.Configs = append(PolicyRoutingConfigSet.Configs, LocalTableRuleConfigs...)
+
+	return nil
 }
 
 func fillLocalRulesFromNode(clientset kubernetes.Interface, nodeName string) error {
 	// Retrieve necessary IP info from the node object.
-	node, err := clientset.CoreV1().Nodes().Get(context.Background(), nodeName, metav1.GetOptions{})
-	if err != nil {
-		return fmt.Errorf("getting node %s: %w", nodeName, err)
+	var node *v1.Node
+	if err := wait.PollUntilContextTimeout(context.Background(), 2*time.Second, 30*time.Second, true, func(context.Context) (bool, error) {
+		var err error
+		node, err = clientset.CoreV1().Nodes().Get(context.Background(), nodeName, metav1.GetOptions{})
+		if err != nil {
+			glog.Errorf("getting node %s: %w", nodeName, err)
```

Comment:
https://google.github.io/styleguide/go/decisions#error-strings

And I think glog.Errorf doesn't take %w?

Reply:
Thanks - fixed now.